com.unity.render-pipelines.high-definition/Editor/Material/MaterialHeaderScope.cs

```diff
@@ -26,6 +26,15 @@ internal struct MaterialHeaderScope : IDisposable
         int oldIndentLevel;
 #endif
 
+        /// <summary>
+        /// Creates a material header scope to display the foldout in the material UI.
+        /// </summary>
+        /// <param name="title">Title of the header.</param>
+        /// <param name="bitExpanded">Bit index to use for the.</param>
```

Comment:
For the what?

Reply:
Hum, this is supposed to be the bit index used to store the state of the header (collapsed/opened) in the Editor Prefs

Reply:
Okay, thanks! How about:

Bit index which specifies the state of the header (whether it is open or collapsed) inside Editor Prefs.